agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -129,8 +129,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     StringBuilder commentDivs = new StringBuilder();
 
-    response.setContentType("application/json");
-
```

Comment:
This change seems unrelated?

Reply:
Sorry, it was unrelated. I was looking at this for example code, and I noticed something wrong with it, and I think it got swept into these changes. I will be more careful not to include anything outside in future PRs.